site/src/components/layout.md

```diff
@@ -0,0 +1,148 @@
+
+The basic boilerplate for a layout is as follows:
+
+```html-no-example
+<html class="layout layout-{nameoflayout}">
+	<body class="layout-body">
+		<header style="layout-body-header">Header</header>
+		<div class="layout-body-hero">Hero</div>
+		<nav class="layout-body-nav">Menu</nav>
+		<main class="layout-body-main">Main</main>
+		<aside class="layout-body-aside">Aside</aside>
+		<footer class="layout-body-footer">Footer</footer>
+	</body>
+</html>
+```
+
+### Apply your own spacing
+
+The `$layout-gap` token is used to generate two separate classes that can be used to apply spacing to containers _within_ each of the major layout areas. Spacing should _not_ be applied to the `layout-body-*` containers themselves, but rather to elements within them - most often a single container that is a direct child of the `layout-body-*` element.
+
+| Class name        | Use case                             | Examples of use case                                                                  |
+| ----------------- | ------------------------------------ | ------------------------------------------------------------------------------------- |
+| `.layout-padding` | Layout areas that are not full width | The menu and aside areas. The main area when not in a single layout                   |
+| `.layout-margin`  | Full width layout areas              | The header, footer, and hero. The main area when it is full width in a single layout. |
+
+Why not use gap? You certainly can apply any [gap atomics](../atomics/gap.md) to the `layout-body` to change spacing as you see fit. However, to provide adequate space to display focus rectangles and to ensure enough space to render dynamic inline elements (like heading anchors), spacing applied to an inner container is still the best choice.
+
+### Responsiveness
+
+The layout components behavior is inextricably bound to Atlas's breakpoints. On narrow widths on all layouts, elements are stacked on top of one another. If you don't want an element to show on a particular screensize, you should either render nothing into it or hide the inner element.
+
+## Available layouts
+
+There are two available layouts.
+
+- [`layout-single`](#layout-single)
+- [`layout-holy-grail`](#holy-grail-layout)
+
+### Layout Single
+
+The `layout-single` class provides a simple stacked layout. It is equivalent to a series of marginless and paddingless stacked block level elements. All elements in a single layout should set their own margin somewhere within their nested containers. Most likely, you'll want to apply `.layout-margin` or some other container-like class to elements within the single layout. This is also the default layout that will render if no other class is provide.
+
+How do I apply it? `layout-holy-grail` to the `layout` element.
```

Comment:
```suggestion
How do I apply it? `layout-single` to the `layout` element.
```

Reply:
Reopening, as this doesn't seem to have been changed, @wibjorn

Reply:
Okay, recommitted this! Not sure what happened there.